e2e/cephfs.go

```diff
@@ -172,549 +194,541 @@ var _ = Describe("cephfs", func() {
+			})
 
-					}
-					subVol = listCephFSSubVolumes(f, fileSystemName, subvolumegroup)
-					if len(subVol) != 0 {
-						msg := fmt.Sprintf("subvolumes %v. subvolume count %d not matching expected count %v", subVol, len(subVol), 0)
-						e2elog.Logf(msg)
-						Fail(msg)
-					}
-				})
+			By("check data persist after recreating pod with same pvc", func() {
+				err := checkDataPersist(pvcPath, appPath, f)
+				if err != nil {
+					e2elog.Failf("failed to check data persist in pvc %v", err)
+				}
+			})
 
-				By("check data persist after recreating pod with same pvc", func() {
-					err := checkDataPersist(pvcPath, appPath, f)
-					if err != nil {
-						Fail(err.Error())
-					}
-				})
+			By("creating a PVC, deleting backing subvolume, and checking successful PV deletion", func() {
+				pvc, err := loadPVC(pvcPath)
+				if pvc == nil {
+					e2elog.Failf("failed to load pvc %v", err)
+				}
+				pvc.Namespace = f.UniqueName
 
-				By("creating a PVC, deleting backing subvolume, and checking successful PV deletion", func() {
-					pvc, err := loadPVC(pvcPath)
-					if pvc == nil {
-						Fail(err.Error())
-					}
-					pvc.Namespace = f.UniqueName
+				err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to create pvc %v", err)
+				}
 
-					err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
-					if err != nil {
-						Fail(err.Error())
-					}
+				err = deleteBackingCephFSVolume(f, pvc)
+				if err != nil {
+					e2elog.Failf("failed to delete cephfs subvolume %v", err)
+				}
 
-					err = deleteBackingCephFSVolume(f, pvc)
-					if err != nil {
-						Fail(err.Error())
-					}
+				err = deletePVCAndValidatePV(f.ClientSet, pvc, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to delete pvc %v", err)
+				}
+			})
 
-					err = deletePVCAndValidatePV(f.ClientSet, pvc, deployTimeout)
-					if err != nil {
-						Fail(err.Error())
-					}
-				})
-
-				By("validate multiple subvolumegroup creation", func() {
-					deleteResource(cephfsExamplePath + "storageclass.yaml")
-					// re-define configmap with information of multiple clusters.
-					subvolgrpInfo := map[string]string{
-						"clusterID-1": "subvolgrp1",
-						"clusterID-2": "subvolgrp2",
-					}
-					createCustomConfigMap(f.ClientSet, cephfsDirPath, subvolgrpInfo)
-					createCephfsStorageClass(f.ClientSet, f, false, "clusterID-1")
-					validatePVCAndAppBinding(pvcPath, appPath, f)
-					deleteResource(cephfsExamplePath + "storageclass.yaml")
-					// verify subvolumegroup creation.
-					err := validateSubvolumegroup(f, "subvolgrp1")
-					if err != nil {
-						Fail(err.Error())
-					}
+			By("validate multiple subvolumegroup creation", func() {
+				err := deleteResource(cephfsExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass %v", err)
+				}
+				// re-define configmap with information of multiple clusters.
```

Comment:
Its good to have a test case for different cluster ID but same subvolumegroup name too. so that we are sure we are not breaking common subvolume name in different clusters.

Reply:
This PR is not changing this one. PRs are welcome to add more testcases